src/costab.c

```diff
@@ -1028,6 +1030,7 @@ static void AddCosetFactor2 (
             leng  = LEN_PLIST(tmp);
             for ( i = 1;  i <= leng;  i++ ) {
                 if ( ! SUM_INTOBJS( sum, ptWord[i], ptFac[i] ) ) {
+                    return 1;
                     ErrorQuit(
```

Comment:
Maybe delete the `ErrorQuit`, and turn its message string into a comment? The same applies about 15 lines below.